app/presenters/two_factor_authentication/sign_in_webauthn_platform_selection_presenter.rb

```diff
@@ -1,5 +1,5 @@
 module TwoFactorAuthentication
-  class WebauthnPlatformSelectionPresenter < SelectionPresenter
+  class SignInWebauthnPlatformSelectionPresenter < SelectionPresenter
     def method
       :webauthn_platform
     end
```

Comment:
I think we can remove checks below to `configuration.blank?`, since I would expect that the configuration should never be blank in the "Sign in" classes. They should be replaced with `false`.